cmd/root_test.go

```diff
@@ -331,11 +327,50 @@ func TestSpecifyingFormatters(t *testing.T) {
 		t.Setenv("TREEFMT_FORMATTERS", "bar,foo")
 
 		treefmt(t,
-			withError(func(err error) {
-				as.Errorf(err, "formatter not found in config: bar")
+			withError(func(as *require.Assertions, err error) {
+				as.ErrorContains(err, "formatter bar not found in config")
 			}),
 		)
 	})
+
+	t.Run("bad names", func(t *testing.T) {
```

Comment:
Nit, feel free to ignore: It seems to me that this is testing 2 things:

1. That all forms of configuration (cli flag, env var, conf file) all work.
2. That we enforce some rules on names.

We should already have tests that confirm 1). I'd argue that testing it again is making this test more verbose than it needs to be, which distracts from the main point of the test.